scripts/js_test.sh

```diff
@@ -22,16 +22,17 @@ export NVM_DIR="$HOME/.nvm"
 [ -s "$NVM_DIR/bash_completion" ] && \. "$NVM_DIR/bash_completion"  # This loads nvm bash_completion
 
 echo "Installing nvm"
-nvm install 16.0
+nvm install 18.20
 
 echo "Using nvm 16"
```

Comment:
```suggestion
echo "Using nvm 18"
```
(or just remove this message)

Shouldn't we just update to the latest version of node now?